packages/tc-web/src/components/tc-collapsible/tc-collapsible.tsx

```
@@ -0,0 +1,65 @@
+            <div class="mt-2 md:block">
+              <button
+                class="cursor-pointer rounded-full border border-uzhblue bg-uzhblue px-4 py-2 font-semibold text-white hover:border-transparent hover:text-white"
+                onClick={this.toggleDetails}
+              >
+                {this.showDetails ? 'Weniger Details' : 'Mehr Details'}
+              </button>
+            </div>
+          </div>
+          <div class="mt-2 flex flex-row flex-wrap gap-2">
+            {parseTags(this.tags).map((tag) => (
+              <div class="bg-gray-200 px-2 py-1 font-medium text-gray-600 md:mt-0">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)